src/test/java/seedu/medibook/testutil/TypicalIndexes.java

```diff
@@ -9,4 +9,6 @@
     public static final Index INDEX_FIRST_PATIENT = Index.fromOneBased(1);
     public static final Index INDEX_SECOND_PATIENT = Index.fromOneBased(2);
     public static final Index INDEX_THIRD_PATIENT = Index.fromOneBased(3);
+    public static final Index INDEX_FIRST_MEDICAL_NOTE = Index.fromOneBased(1);
+    public static final Index INDEX_SECOND_MEDICAL_NOTE = Index.fromOneBased(2);
```

Comment:
If this class is solely to create static Index objects, does there need to be separate objects to use for patient and medical note tests? Since they are pointing to similar objects anyways. Additionally, "index first" sounds a little strange. Maybe "index one" or "first index"? Not sure what makes sense.

Reply:
public static final Index INDEX_FIRST_PATIENT = Index.fromOneBased(1);
public static final Index INDEX_SECOND_PATIENT = Index.fromOneBased(2);
public static final Index INDEX_THIRD_PATIENT = Index.fromOneBased(3);
are from AB-3. 

Should I just rename all to INDEX_FIRST, INDEX_SECOND, INDEX_THIRD?
INDEX_ONE may be misleading since there is one-based and zero-based indices.
The variables should be named with the same first term "INDEX" as per naming convention.